geode-core/src/test/java/org/apache/geode/test/dunit/cache/internal/JUnit4CacheTestCase.java

```diff
@@ -292,17 +258,8 @@ public final ClientCache getClientCache(final ClientCacheFactory factory) {
       final GemFireCacheImpl gemFireCache = GemFireCacheImpl.getInstance();
       if (gemFireCache != null && !gemFireCache.isClosed()
           && gemFireCache.getCancelCriterion().isCancelInProgress()) {
-        Wait.waitForCriterion(new WaitCriterion() { // TODO: replace with Awaitility
-          @Override
-          public boolean done() {
-            return gemFireCache.isClosed();
-          }
-
-          @Override
-          public String description() {
-            return "waiting for cache to close";
-          }
-        }, 30 * 1000, 300, true);
+        Awaitility.await("waiting for cache to close").atMost(30, TimeUnit.SECONDS)
```

Comment:
I know this was here before, but waiting for 30 seconds in a Unit test strikes me as unacceptable. Do we know if this is actually required?

Reply:
It looks like it's waiting for the cache to shut down. I'm not too worried, mostly because:
* I'm lazy
* I think the cache will shut down quickly?
* Despite the name, this is the base case only for DUnit tests.